### PR TITLE
Installation now based on Archive files

### PR DESCRIPTION
**Pull request by ReinhardKeil**

### CODE REVIEW

File: tools/toolbox/README.md [hunk 1]
```diff
@@ -13,4 +13,7 @@ Tool               | Binary         | Description
 :------------------|:---------------|:-------------------------------------------------
 Package Installer  | cpackget       | Install packs into local environment
 Project Manager    | csolution      | Validate multiproject solutions and generate CPRJs
-Build Manager      | cbuildgen      | Generate CMakeLists and invoke CMake/Ninja
+Build Invocation   | cbuild         | Start the build process via Build Generator
+Build Generator    | cbuildgen      | Generate CMakeLists and start CMake/
+Ninja
+Pack Verification  | packchk        | Sematic validation of a software pack
```
Comment (brondani) [inline tools/toolbox/README.md:19]:
```suggestion
Pack Verification  | packchk        | Semantic validation of a software pack
```

File: tools/toolbox/docs/installation.md [hunk 2]
```diff
@@ -50,35 +50,17 @@ The CMSIS-Toolbox works with the following toolchains. Install one or more toolc
...
-./cmsis-toolbox.sh -x ./ctools
-```
+> **Note:**
+> 
+> `cmsis-toolbox.sh` is provided for legacy reasons, but may be depreciated in future versions.
 
 ## Configuration
 
-For the manual installation described above the configuration setup is as follows:
+It is required to customize the installation for the actual setup of your development enviornment as described in the following:
```
Comment (brondani) [inline tools/toolbox/docs/installation.md:63]:
```suggestion
It is required to customize the installation for the actual setup of your development environment as described in the following:
```

File: tools/toolbox/docs/installation.md [hunk 3]
```diff
@@ -50,35 +50,17 @@ The CMSIS-Toolbox works with the following toolchains. Install one or more toolc
...
-  -x [<dir>]   : Extract full content into optional directory
-```
-
-Below the manual setup of the CMSIS-Toolbox is explained. The command below installs the tools into the directory `./ctools`.
-```txt
-./cmsis-toolbox.sh -x ./ctools
-```
+> **Note:**
+> 
+> `cmsis-toolbox.sh` is provided for legacy reasons, but may be depreciated in future versions.
```
Comment (brondani) [inline tools/toolbox/docs/installation.md:59]:
```suggestion
> `cmsis-toolbox.sh` is provided for legacy reasons, but may be deprecated in future versions.
```

File: tools/toolbox/docs/installation.md [hunk 4]
```diff
@@ -50,35 +50,17 @@ The CMSIS-Toolbox works with the following toolchains. Install one or more toolc
...
 
 ## Configuration
 
-For the manual installation described above the configuration setup is as follows:
+It is required to customize the installation for the actual setup of your development enviornment as described in the following:
 
 ### ./etc/setup
```
Comment (brondani) [inline tools/toolbox/docs/installation.md:65]: In the new archives the `setup` file and everything related to `bash` is not needed, so I would suggest to remove the text between lines 65 and 82.

In lines 89 and 90 I would also inform `CMSIS_PACK_ROOT` and `CMSIS_COMPILER_ROOT` are now optional. If not specified their default values are respectively `%LOCALAPPDATA%/Arm/Packs` and the CMSIS-Toolbox `etc` directory.

In line 93 the correct path to *.cmake files should be just `./etc/*.cmake`

Comment (ReinhardKeil) [reply tools/toolbox/docs/installation.md:65]: OK, I will address this as next step.  I believe we need instructions for installation on each OS.

**Review by brondani — APPROVED** [review]

**Pull request merged**